Post/models.py

```diff
@@ -7,3 +7,8 @@ class post(models.Model):
     nameOfLocation = models.CharField(max_length=100)
     photoURL = models.TextField()
     Description = models.TextField()
+
+    def __str__(self):
+        txt = "{0} traveled {1} and wrote: {2}"
+
+        return txt.format(self.nameOfPoster, self.nameOfLocation, self.Description)
```

Comment:
Here is a shorter form: `return f'{self.nameOfPoster} traveled {self.nameOfLocation} and wrote: {self.Description}'`

Reply:
Thanks! I change it.